### PR TITLE
Used foundation to style the battle view.

### PR DESCRIPTION
**Pull request by supdood**
-Replies are no indicated with offsets.
-Sibling comments alternate their background colors.
-The hamburger menu button is now of fixed positioning when scrolling.

### CODE REVIEW

File: app/assets/stylesheets/battles.scss [hunk 1]
```diff
@@ -6,6 +6,55 @@
...
+}
+
+button.button:hover {
+  background-color: lightslategray;
+  color: black;
 }
+
+.js-off-canvas-exit .is-visible {
+  //position: fixed;
+}
```
Comment (J3RN) [inline app/assets/stylesheets/battles.scss]: Since this is commented out, can it be removed?

**Comment by J3RN** [comment]
Overall, this looks dope :+1:

File: app/assets/stylesheets/battles.scss [hunk 2]
```diff
@@ -6,6 +6,55 @@
...
+  background-color: #BFCCC9;
+}
+
+.color-1 {
+  background-color: #EBFBF8;
+}
+
 .avatar {
-  width: 50px;
+  width: 70px;
```
Comment (BFGeorge9000) [inline app/assets/stylesheets/battles.scss]: 1.) Looks like we have some weirdness going on with the indentation in this class
2.) If possible, we should avoid styling pixel values.  Let's try to rely on Foundation classes to size our elements (A `small-1` sized div might be about the same size you're hoping to see in your layout and will be much more flexible for a responsive design)

Comment (supdood) [inline app/assets/stylesheets/battles.scss]: Got it!

File: app/assets/stylesheets/battles.scss [hunk 3]
```diff
@@ -6,6 +6,51 @@
...
+
+.color-1 {
+  background-color: #EBFBF8;
+}
+
 .avatar {
-  width: 50px;
+  margin-top: 10px;
+	margin-bottom: 5px;
+	border-radius: 50%;
```
Comment (J3RN) [inline app/assets/stylesheets/battles.scss]: Out of curiosity, what is going on with the tabbing here?

Comment (supdood) [inline app/assets/stylesheets/battles.scss]: @J3RN I thought that might happen somewhere.  So apparently when I changed the brackets setting, it only made tabs look like 2 spaces even though they were still 4.  I had to change a setting to make them actually act as 2 spaces.  So those further tabbed lines are from before I fixed that.

**Comment by supdood** [comment]
Conflicts? :fearful:

**Comment by BFGeorge9000** [comment]
:+1:

**Pull request merged**